doc/requirements/requirements.md

```diff
@@ -0,0 +1,306 @@
+IC003 | Implementation Language.C++.API Preference | 1 | Developers SHALL prefer standard C++, then Boost, then custom code, in that order. | Boost may be used if equivalent functionality is not already available in the C++ standard library
+IC004 | Implementation Language.C++.Supported Compilers.g++ | 1 | The Navigation System code SHALL compile with gcc 5.4 or newer
+IC004 | Implementation Language.C++.Supported Compilers.Clang | 1 | The Navigation System code SHALL compile with Clang, version *x*
+IC004 | Implementation Language.C++.Supported Compilers.Intel C++ Compiler | 1 | The Navigation System code SHOULD compile with the Intel C++ Compiler, version *x* | Could be useful for optimization purposes
+IC005 | Implementation Language.Python.Version | 1 | Any Python code developed for the Navigation System MUST use Python 3
+IC006 | Implementation Language.GUI | 1 | Any GUIs developed as part of the Navigation System MUST use the Qt library, via C++ or Python (PyQt) | *Which version?*
+IC006 | Implementation Language.GUI.QML | 1 | Any GUIs developed as part of the Navigation System MAY use QML
+
+## 2.2 Development Tools
+
+This section specifies the development tools and associated targets that will be used for the project.
+
+Id | Handle | Priority | Description | Notes
+-- | ------ | -------- | ----------- | -----
+DEV001 | Tools.ROS2.Version | 1 | The Navigation System WILL be developed against the latest stable version of the ROS 2 stack | *What is the current latest version?*
+DEV002 | Tools.Unit Test.Library | 1 | *TODO: GTest and GMock?*
+DEV003 | Tools.Unit Test.Module Requirements | 1 | *TODO: What are the module level requirements?*
+DEV004 | Tools.Static Analysis | 1 | *TODO: What is the static analysis tool to use?*
+DEV005 | Tools.Static Analysis.Quality Targets | 1 | *TODO: What are the static analysis targets?*
+DEV006 | Tools.Dynamic Analysis | 1 | *TODO: What dynamic analysis tool(s) to use?*
+DEV007 | Tools.Dynamic Analysis.Code Coverage Targets | 1 | *TODO: What are the code coverage targets?*
+DEV008 | Tools.Dynamic Analysis.Memory Checking | 1 | *TODO: What is/are the memory checking tool(s) to use?*
+
+## 2.3 Target Platforms
+
+The Navigation System will run on the latest versions of the operating systems supported by the core ROS 2 code.
+
+Id | Handle | Priority | Description | Notes
+-- | ------ | -------- | ----------- | -----
+PLT001 | Target Platforms.Operating Systems.Ubuntu | 1 | The Navigation System MUST support Ubuntu Desktop 16.04 and Ubuntu Desktop 18.04
+PLT002 | Target Platforms.Operating Systems.MacOS | 1 | The Navigation System MUST support MacOS 10.13 (High Sierra) and MacOS 10.14 (Mohave)
+PLT003 | Target Platforms.Operating Systems.Windows | 1 | The Navigation System MUST support Windows 10 Professional
+PLT004 | Target Platforms.Operating Systems.Clear Linux | 1 | The Navigation System SHOULD support the latest version of Intel's Clear Linux distribution | What is the latest version number?
```

Comment:
Clear linux doesn't have a concept of a "stable" release, they follow more of a continuous deployment model. However, we'll be able to put in a version number at a future date.

Reply:
Ok, I'll update.